### PR TITLE
fix(package): add new route for package & module to latest version

### PR DESCRIPTION
**Pull request by revathskumar**
Adding routes for `/latest/`

* /:author/:project/latest/module/:module
* /:author/:project/latest/overview

### CODE REVIEW

File: src/routes/package.js [hunk 1]
```diff
@@ -277,6 +187,118 @@ async function prepareModuleDocumentation(moduleInfo) {
   return [intro].concat(parts);
 }
 
+async function moduleOverview(ctx, next) {
+  const { author, project } = ctx.params;
+  const packageName = packageNameFromCtx(ctx);
+  let version = ctx.params.version;
+  if (!version) {
+    version = await dbPackage.latestVersion(packageName);
+  }
```
Comment (revathskumar) [inline src/routes/package.js:196]: relevant change.

File: src/routes/package.js [hunk 2]
```diff
@@ -277,6 +187,118 @@ async function prepareModuleDocumentation(moduleInfo) {
...
+}
+
+async function packageOverview(ctx, next) {
+  const author = ctx.params.author;
+  const project = ctx.params.project;
+  const packageName = packageNameFromCtx(ctx);
+  let version = ctx.params.version;
+  if (!version) {
+    version = await dbPackage.latestVersion(packageName);
+  }
```
Comment (revathskumar) [inline src/routes/package.js:253]: relevant change.

**Comment by revathskumar** [comment]
Let me know if there is anything more need to handled.

**Comment by robinheghan** [comment]
This looks good to me, thank you for making this change!

Before merging, would you mind adding your name and username to the `CONTRIBUTORS` file?

**Comment by revathskumar** [comment]
@robinheghan Done. ([deb7f5a](https://github.com/gren-lang/package.gren-lang.org/pull/23/commits/deb7f5a1fd8b0840355be7affb819a2e1a9981e5))

**Comment by robinheghan** [comment]
> @robinheghan Once this merged and deployed. Should I update the [links in this PR](https://github.com/gren-lang/parser/pull/15) to point to `/latest/`

That would be great, yeah :)

**Pull request merged**